contracts/libs/utils/TypeHelper.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "../../interfaces/gov/IGovPool.sol";
+
+library TypeHelper {
```

Comment:
Can `GovUserKeeperView` be optimized with this?

Reply:
done